### PR TITLE
Bug 1849069 - Real name change doesn't work if "New email address" is pre-filled with different email by password manager

### PR DESCRIPTION
**Pull request by LLyaudet**
To make it case insensitive like it is the case for login. See https://bugzilla.mozilla.org/show_bug.cgi?id=1849069

### CODE REVIEW

**Review by dklawren — CHANGES_REQUESTED** [review]
This will work and thanks for the pull request. We also want to add the autocomplete="off" to the new email form field as mentioned in the bug. 

https://github.com/mozilla-bteam/bmo/blob/master/template/en/default/account/prefs/account.html.tmpl#L99

**Comment by LLyaudet** [comment]
I just committed another commit for the autocomplete="off". Tell me if there is something else to modify :)

File: template/en/default/account/prefs/account.html.tmpl [hunk 1]
```diff
@@ -107,7 +107,7 @@
           <th align="right">Current password:</th>
           <td>
             <input type="hidden" name="old_login" value="[% user.login FILTER html %]">
-            <input autocomplete="current-password" type="password" name="old_password" id="old_password">
+            <input autocomplete="off" type="password" name="old_password" id="old_password">
```
Comment (dklawren) [inline template/en/default/account/prefs/account.html.tmpl:110]: Actually after seeing that we already do autocomplete for current password, let's change this back to `current-password` as looking this up that is correct already.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

**Review by dklawren — CHANGES_REQUESTED** [review]
After that this should be all good.

File: template/en/default/account/prefs/account.html.tmpl (hunk 1 above)
Comment (LLyaudet) [reply template/en/default/account/prefs/account.html.tmpl]: You are right. Both were syntactically correct but keeping "current-password" value here is a feature and yields no bug.

**Comment by LLyaudet** [comment]
You're welcome :)

**Pull request merged**